src/views/Error.vue

```diff
@@ -14,6 +14,9 @@
 
       <!-- TODO
       remove hardcoded link -> either parameterize it, or enter a link to youtube -->
+      <div class="lead_text">
+        <p>Plio को कैसे इस्तेमाल करना है, जानने के लिए ये video देखें </p>
+      </div>
```

Comment:
Added text over the tutorial video embed

Reply:
Gave some feedback for the text